packages/actors_api/src/actors/Attester.spec.ts

```diff
@@ -26,7 +27,7 @@ describe('Attester', () => {
     claimer = Identity.buildFromURI('//Bob')
 
     rawCType = {
-      $id: 'kilt:ctype:0x1',
+      $id: Crypto.hashStr('kilt:ctype:0x1'),
```

Comment:
why does this need to be hashed?

Reply:
It doesn't :)

Reply:
It doesn't I have updated it accordingly.